config_gen.py

```diff
@@ -303,6 +289,42 @@ def run(cmd, *args, **kwargs):
         print("\nRunning make...")
         run(make_args, env=env, **proc_opts)
 
+    elif(os.path.exists(os.path.join(project_dir, "Make/options"))):
+        # Added for supporting wmake (OpenFOAM build system) 2016/03/18
```

Comment:
You don't need to include the date here, because git will store this information for us - `wmake (OpenFOAM build system)` is all that's needed.